library/Businessprocess/Renderer/TileRenderer/NodeTile.php

```diff
@@ -338,7 +338,7 @@ protected function addActionLinks()
             }
         }
 
-        if ($this->renderer->getBusinessProcess()->getMetadata()->canModify()) {
+        if ($this->renderer->getBusinessProcess()->getMetadata()->canModify() && $this->node instanceof ImportedNode) {
```

Comment:
Why is this required now?

Reply:
As delete button is moved to config form the remove icon `x` is removed from the node tiles except for imported nodes which you cannot edit.